Sweeper/AppDelegate.swift

```diff
@@ -18,13 +18,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-
+        
         let configuration = ParseClientConfiguration {
             $0.applicationId = "com.team11.Sweeper"
             $0.server = "http://165.227.6.232:1337/parse"
+            $0.isLocalDatastoreEnabled = true
         }
+        
         Parse.initialize(with: configuration)
         
+        User.getStoredUser(completion: { (user: User?) in
```

Comment:
can we check User.currentuser like we did for Twitter? I think that's a better pattern to use.

Reply:
Yea but this is a blocking call for Parse, so it's better to do it in a closure. We'll only call this once in appdelegate and from then on we can access via User.currentUser (once it's fetched).

Reply:
Swift gives errors about blocking the main thread if we don't do it in a callback.

Reply:
Gotcha, sounds good!